README.rst

```diff
@@ -51,15 +46,68 @@ NeMo's works with:
 1) Python 3.6 or 3.7
 2) Pytorch 1.6 or above
 
-Installation
-~~~~~~~~~~~~
-``pip install nemo_toolkit[all]==version``
-
-We recommend using NVIDIA's PyTorch container
+Docker container:
+~~~~~~~~~~~~~~~~~
+We recommend using NVIDIA's PyTorch container version 20.06-py3 with NeMo's main branch.
 
 .. code-block:: bash
 
     docker run --gpus all -it --rm -v <nemo_github_folder>:/NeMo --shm-size=8g \
     -p 8888:8888 -p 6006:6006 --ulimit memlock=-1 --ulimit \
     stack=67108864 nvcr.io/nvidia/pytorch:20.06-py3
 
+Installation
+~~~~~~~~~~~~
+Once requirements are satisfied (or you are inside NVIDIA docker container), simply install using pip:
+
+``pip install nemo_toolkit[all]==version``
+
+* ``pip install nemo_toolkit[all]`` - latest released version (currently 0.11.0)
+* ``python -m pip install git+https://github.com/NVIDIA/NeMo.git@{BRANCH}#egg=nemo_toolkit[nlp]`` - where {BRANCH} should be replaced with the branch you want. This is recommended route if you are testing out the latest WIP version of NeMo.
+
+
+Documentation
+-------------
+* `Main <https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/candidate/>`_
+* `Latest released (0.11.0) <https://docs.nvidia.com/deeplearning/nemo/developer_guide/en/v0.11.0/>`_
+
+Tutorials
+---------
+The best way to get started with NeMo is to checkout one of our tutorials.
+
+Most NeMo tutorials can be run on `Google's Collab <https://colab.research.google.com/notebooks/intro.ipynb>`_.
+
+Steps to run tutorial:
+
+1. Go to `Google's Collab <https://colab.research.google.com/notebooks/intro.ipynb>`_.
+2. Import tutorial notebook from GitHub (File -> Upload Notebook -> "GITHUB" tab -> copy/paste GitHub URL)
+3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select "GPU" for hardware accelerator)
+
+.. list-table:: NeMo tutorials
+   :widths: 15 25 25
+   :header-rows: 1
+
+   * - Domain
+     - Title
+     - GitHub URL
+   * - ASR
+     - ASR with NeMo
+     - `https://github.com/NVIDIA/NeMo/blob/candidate/tutorials/asr/01_ASR_with_NeMo.ipynb <https://github.com/NVIDIA/NeMo/blob/candidate/tutorials/asr/01_ASR_with_NeMo.ipynb>`_
```

Comment:
If we are using links here, we can just link directly to colab with the following syntax : https://colab.research.google.com/github/NVIDIA/NeMo/blob/candidate/tutorials/asr/01_ASR_with_NeMo.ipynb

Reply:
You'll need to replace "candidate" portion with master or main later

Reply:
not sure about colab though... @okuchaiev's decision.

Reply:
Yup these are just suggestions. The block is the name Collab -> Colab

Reply:
But I kinda use it a lot in my own libraries cause it reduces friction by a lot

Reply:
For example in ramp up notebooks from - https://github.com/titu1994/tfdiffeq#notebooks-to-get-started

Reply:
thanks - fixed the typo. changed links to colab and removed some text.

We will have some local-only notebook though - e.g. when microphone is needed

Reply:
Yup those need to be pure Jupyter Notebooks.